201-managed-application-usingmsi-for-management/artifacts/ManagedAppZip/createUiDefinition.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/0.1.2-preview/CreateUIDefinition.MultiVm.json#",
+  "handler": "Microsoft.Azure.CreateUIDef",
+  "version": "0.1.2-preview",
+  "parameters": {
+    "basics": [],
+    "steps": [
+      {
+        "name": "applicationSettings",
+        "label": "Application settings",
+        "subLabel": {
+          "preValidation": "Configure the application settings",
+          "postValidation": "Done"
+        },
+        "bladeTitle": "Application settings",
+        "elements": [
+          {
+            "name": "storageAccount",
+            "type": "Microsoft.Common.TextBox",
```

Comment:
use the storageAccount control for this - be sure to set hideExisting to true and output all the properties of the control

Reply:
Fixed